source/Calamari.Tests.Shared/CommandTestBuilder.cs

```diff
@@ -28,14 +45,62 @@ public static CommandTestBuilder<TCalamari> Create<TCalamari>(string command)
             return new CommandTestBuilder<TCalamari>(typeof(TCommand).GetCustomAttribute<CommandAttribute>().Name);
         }
 
-        public static CommandTestBuilderContext WithPackage(this CommandTestBuilderContext context, string path)
+        public static CommandTestBuilderContext WithFilesToCopy(this CommandTestBuilderContext context, string path)
         {
-            context.Variables.Add(KnownVariables.OriginalPackageDirectoryPath, Path.GetDirectoryName(path));
+            if (File.Exists(path))
+            {
+                context.Variables.Add(KnownVariables.OriginalPackageDirectoryPath, Path.GetDirectoryName(path));
+            }
+            else
+            {
+                context.Variables.Add(KnownVariables.OriginalPackageDirectoryPath, path);
+            }
+
             context.Variables.Add("Octopus.Action.Package.PackageId", path);
```

Comment:
Setting the package ID to the path is misleading - do we have a better alternative here?

Reply:
it is an internal concern, do we care?

Reply:
It causes confusion, can we just not set it? If we have to set it, then set it to an empty string or something like `"Dummy value, not used"`

Reply:
but we use that value, that is how we copy the files